### PR TITLE
Blogpost about  Regions

### PR DESCRIPTION
**Pull request by julien-c**
(i'll also add similar doc to the hub-docs)

### CODE REVIEW

**Review by VioletteLepercq — COMMENTED** [review]
nice !! (changed sovereingty for sovereignty)

File: regions.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Introducing Storage Regions on the HF Hub
+
+As part of our [Enterprise Hub](https://huggingface.co/enterprise) plan, we recently relased support for **Storage Regions**.
```
Comment (pcuenca) [inline regions.md:3]:
```suggestion
As part of our [Enterprise Hub](https://huggingface.co/enterprise) plan, we recently released support for **Storage Regions**.
```

File: regions.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Any repo (model or dataset) stored in a non-default location will display its Region directly as a tag. That way your organization's members can see at a glance where repos are located.
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/tag-on-repo.png)
+
+
+## Regulatory and legal compliance
+
+In many regulated industries, you have a requirement to store your data in a specific area.
+If you are an Enterprise Hub customer and have questions about this, please get in touch!
```
Comment (pcuenca) [inline regions.md:41]: Is there a link we can add to `get in touch`?

File: regions.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Performance
+
+Storing your models or your datasets closer to your team and infrastructure also means significantly improved performance, for both uploads and downloads.
+
+This makes a big difference considering model weights and dataset files are usually very large.
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/storage-regions/upload-speed.png)
+
+As an example, if you are located in Europe, you can expect to see ~4-5x faster upload and download speeds vs. repositories stored in the US.
```
Comment (pcuenca) [inline regions.md:51]: I may be overthinking it, but this may sound as customers in the US having a speed disadvantage in general. Maybe rephrase slightly?

```suggestion
As an example, if you are located in Europe and store your repositories in the EU region, you can expect to see ~4-5x faster upload and download speeds vs. if they were stored in the US.
```

**Comment by jeffboudier** [comment]
Would it make sense to specifically call out GDPR compliance for EU companies? Both as content as for SEO.

**Comment by julien-c** [comment]
@jeffboudier good idea! can you suggest a short sentence?

File: regions.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Regions let you decide where your org's models and datasets will be stored. This has two main benefits, which we'll briefly go over in this blog post:
+- **Regulatory and legal compliance**, and more generally, better digital sovereignty
+- **Performance** (improved download and upload speeds and latency)
+
+Currently we support the following regions:
+- US 🇺🇸
+- EU 🇪🇺
+- coming soon: Asia-Pacific 🌏
+
+But first, let's see how to setup this feature in your organization's settings 🔥
```
Comment (jeffboudier) [inline regions.md:14]:
```suggestion

For companies in the EU, that means you can use the Hub to build ML in a GDPR compliant way: with datasets, models and inference endpoints all stored within EU data centers.

But first, let's see how to setup this feature in your organization's settings 🔥
```

**Review by julien-c — COMMENTED** [review]
thanks for the reviews, i'll publish today i think
